packages/substrate-client/src/client/index.ts

```diff
@@ -1,2 +1,3 @@
 export * from "./RpcError"
 export * from "./createClient"
+export * from "./DisconnectedError"
```

Comment:
Unless I misunderstood something, the `DisconnectedError` is an internal implementation detail that shouldn't leak through the public API, correct?

If that's the case, then we should make sure to change the `src/index.tx` file in order to prevent the `DisconnectedError` from being part of the public API.

Reply:
I'm just now writing the same test for transaction and we might actually need to expose it, since transaction doesn't have `DisjointError`.

Maybe it's not an issue there, since there are no promises, only a onSuccess/onError callback.